src/main/kotlin/org/js/vento/plugin/VentoTypes.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 Óscar Otero
+ * All rights reserved.
```

Comment:
Since you did all the work, I don't think my name deserves to appear in the copyright. Feel free replace it or at least add your name there :)